src/content/governance/index.md

```diff
@@ -0,0 +1,151 @@
+
+> Potential outcomes of this stage are:
+
+> - The EIP will be considered for a future network upgrade
+> - Technical changes will be requested
+> - It may be rejected if it is not a priority or the improvement is not large enough relative to the development effort.
+
+3. **[Optional] Gather Community Consensus for your EIP**: some EIPs are purely technical changes with little room for debate, but others can be more contentious. If the EIP is in the latter category, Protocol Developers will likely want to see strong evidence of overwhelming community support before proceeding with it. There is no "one size fits all" approach here, as each EIP is unique.
+
+4. **EIP Included in Network Upgrade**: assuming the EIP is approved, tested and implemented, it gets scheduled as part of a network upgrade. Given the high coordination costs of network upgrades (everyone needs to upgrade simultaneously), EIPs are generally bundled together in upgrades.
+
+5. **Network Upgrade Activated**: after the network upgrade is activated, the EIP will be live on the Ethereum network. _Note: network upgrades are usually activated on testnets before being activated on the Ethereum Mainnet._
+
+This flow, while very simplified, gives an overview of the significant stages for a protocol change to be activated on Ethereum. Now, let's look at the informal factors at play during this process.
+
+## Informal Process {#informal-process}
+
+### Understanding Prior Work {#prior-work}
+
+EIP Champions should familiarise themselves with prior work and proposals before having an EIP, which can be seriously considered for deployment on the Ethereum Mainnet. This way, the EIP hopefully brings something new which hasn't been rejected before. The three main places to research this are the [EIP repository](https://github.com/ethereum/eips), [Ethereum Magicians](https://www.ethereum-magicians.org/) and [ethresear.ch](https://www.ethresear.ch/).
+
+### Working Groups {#working-groups}
+
+It is unrealistic to expect that the initial EIP will be included on the Ethereum Mainnet without change for more significant initiatives. In practice, these will require EIP Champions to work with a subset of Protocol Developers to specify, implement, test, iterate on and finalize their proposal. Historically, these working groups have involved several months (and sometimes years!) of work. Similarly, EIP Champions for such changes should involve relevant Application/Tooling Developers early in their efforts to gather end-user feedback and mitigate any deployment risks.
+
+### Community Consensus {#community-consensus}
+
+Unfortunately, there is no clear playbook on how to handle contentious proposals. Because they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs who are contentious amongst various stakeholder groups.
```

Comment:
```suggestion
Unfortunately, there is no clear playbook on how to handle contentious proposals. Since they have no way to force people to adopt network upgrades, Protocol Developers will generally avoid including EIPs that are contentious amongst various stakeholder groups.
```